en/docs/integrations/integration-concepts.md

```diff
@@ -1,57 +1,54 @@
 # Integration Concepts
-Before you start building integrations with Choreo, there are basic concepts that you need to understand. This section walks you through all the integration concepts you need to know. 
- 
+Choreo integrations allow you to create enterprise integrations to automate workflows in a user-friendly low-code environment. Here are some basic concepts which will help you understand Choreo integrations as you develop your integrations in the [Choreo Console]({{choreo_console}}).
+  
 ## Integration
-Integration in Choreo is the process of connecting APIs, events, and streams to function as a single unit that can share information.
-
-## Integration Template
-An integration template in Choreo is a prebuilt sample designed to address a common integration requirement. Choreo provides a comprehensive set of integration templates for you to select and use depending on your requirement.
+Integration in Choreo is a process that connects APIs, events, and streams allowing these to function as a single unit. Choreo provides an integration template which is a pre-built sample designed to address a common integration requirement. Choreo includes a set of comprehensive integration templates for you to select and use.
 
 ## Trigger
-A trigger in Choreo integrations is an event or an action that can cause an integration to start executing.
+A trigger in Choreo integrations is an event or an action that causes a Choreo integration to start executing.
 
 Choreo supports the following trigger types:
 
- - **Schedule**: To trigger an integration based on a schedule that you set.
- - **Manual**: To run and test an integration manually.
- - **Calendar**: To trigger an integration based on events in a Google Calendar.
- - **GitHub**: To trigger an integration based on GitHub events and actions.
- - **Salesforce**: To trigger an integration based on topics in a Salesforce account. 
+ - **Schedule**: Triggers execution according to a given schedule.
+ - **Manual**: Triggers execution manually.
+ - **Calendar**: Triggers execution based on events in a Google Calendar.
+ - **GitHub**: Triggers execution based on GitHub events and actions.
+ - **Salesforce**: Triggers execution based on Salesforce events. 
 
-## API Call
-An API call in Choreo integrations allows you to connect to and interact with an external service through your integration. You can use an API call to interact with generic protocol-based APIs and application-specific APIs.
+## API call
+An API call in Choreo allows you to connect to an external service through your Choreo integration. You can use an API call to interact with generic protocol-based APIs and application-specific APIs.
 
-Following are the connections you can use to interact with generic protocol-based APIs:
+The following lists the connections you can use to interact with generic protocol-based APIs:
```

Comment:
What is the reason to change this line only in this file and not in service-conncepts.md page?

Reply:
They are both grammatically correct sentences, giving out the same meaning. However, changed all sentences to match.